lib/commands/debug.ts

```diff
@@ -22,16 +22,16 @@
 		let debugData = this.$debugDataService.createDebugData(this.$projectData, this.$options);
 
 		await this.$platformService.trackProjectType(this.$projectData);
+		const devices = this.$devicesService.getDeviceInstances();
```

Comment:
why is this line moved? By definition the `const`s should be declared where they are used, not before that.